lib/colorls/flags.rb

```diff
@@ -90,9 +99,11 @@ def fetch_sort_opts
     def report_incompatible_flags
       return '' if @opts[:show].nil? || @opts[:sort].nil?
 
-      return 'Restrain from using -t (--tree) and -r (--report) flags together.' if @opts[:tree] && @opts[:report]
-
-      return 'Restrain from using -t (--tree) and -a (--all) flags together.' if @opts[:tree] && @opts[:all]
+      if @opts[:tree]
+        return 'Restrain from using -t (--tree) and -r (--report) flags together.' if @opts[:report]
+        return 'Restrain from using -t (--tree) and -l (--long) flags together.' if @opts[:long]
+        return 'Restrain from using -t (--tree) and -a (--all) flags together.' if @opts[:all]
+      end
```

Comment:
The previous structure seemed more simpler to read - can you add the new check between -t and -l flags with that structure?

```ruby
return 'Restrain from using -t (--tree) and -l (--long) flags together.' if @opts[:tree] && @opts[:long]
```

Reply:
Rubocop considered the previous structure too complex, try it yourself and you will see.

Reply:
Hmm, maybe an array form to iterate would be better? Something like,

```ruby
[
  ['-t (--tree)', @opts[:tree], '-r (--report)', @opts[:report],
  ['-t (--tree)', @opts[:tree], '-l (--long)', @opts[:long],
  ['-t (--tree)', @opts[:tree], '-a (--all)', @opts[:all]
].each do |flag_1, flag_1_given, flag_2, flag_2_given|
  return "Restrain from using #{flag_1} and #{flag_2} flags together." if flag_1_given && flag_2_given
end
```

If this is also causing Rubocop offense, you can revert to your previous change itself.